hivemq-edge/src/frontend/src/api/hooks/useConnection/useGetBridgesStatus.spec.tsx

```diff
@@ -0,0 +1,41 @@
+import { expect } from 'vitest'
+import { renderHook, waitFor } from '@testing-library/react'
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import { MemoryRouter } from 'react-router-dom'
+
+import { server } from '@/__test-utils__/msw/mockServer.ts'
+import { AuthProvider } from '@/modules/Auth/AuthProvider.tsx'
+
+import { handlers } from './__handlers__'
+import { useGetBridgesStatus } from '@/api/hooks/useConnection/useGetBridgesStatus.ts'
+
+const wrapper: React.JSXElementConstructor<{ children: React.ReactElement }> = ({ children }) => (
```

Comment:
question: okay, yeah you might want to export and reimport it here :D

Reply:
There are only 27 of them :-) 
Fixing it